src/app_service/service/community_tokens/service.nim

```diff
@@ -175,6 +189,23 @@ QtObject:
         let addressFrom = self.contractOwner(collectibleAndAmount.communityToken.chainId, collectibleAndAmount.communityToken.address)
         let txData = TransactionDataDto(source: parseAddress(addressFrom)) #TODO estimate fee in UI
         let response = tokens_backend.mintTo(collectibleAndAmount.communityToken.chainId, collectibleAndAmount.communityToken.address, %txData, password, walletAddresses, collectibleAndAmount.amount)
-        echo "!!! Transaction hash ", response.result.getStr()
+        let transactionHash = response.result.getStr()
+        echo "!!! Transaction hash ", transactionHash
```

Comment:
Do we still need this? We can always print it in a `debug` instead if we want it.